src/app/shared/data-input/data-input.component.html

```diff
@@ -3,7 +3,7 @@
   <address-data-input *ngSwitchCase="constraintType.Address"
                       [focus]="focus"
                       [readonly]="readonly"
-                      [skipValidation]="skipValidation"
+                      [skipValidation]="configuration.skipValidation"
```

Comment:
I think you can pass new interface everywhere. It will be necessary in future change.

Reply:
I don't know, the other properties do not make any sense for the component and it would create a fake impression that they make a difference or influence something...